micromasters/urls.py

```diff
@@ -50,8 +51,9 @@
     url(r'^api/v0/financial_aid_request/$', FinancialAidRequestView.as_view(), name='financial_aid_request'),
     url(r'^api/v0/financial_aid_action/(?P<financial_aid_id>[\d]+)/$', FinancialAidActionView.as_view(),
         name='financial_aid_action'),
-    url(r'^api/v0/financial_aid_course_price/(?P<program_id>[\d]+)/$',
-        GetLearnerPriceForCourseView.as_view(), name='financial_aid_course_price'),
+    url(r'^api/v0/course_prices/$', CoursePriceListView.as_view(), name='course_price_api'),
+    url(r'^api/v0/course_prices/(?P<program_id>[\d]+)/$',
+        CoursePriceDetailView.as_view(), name='course_price_api'),
```

Comment:
The convention for DRF views is to append a `-detail` to the detail URL name and a `-list` to the list name. Can you do that here as well?